mode/vue/vue.js

```diff
@@ -0,0 +1,193 @@
+// CodeMirror, copyright (c) by Marijn Haverbeke and others
+// Distributed under an MIT license: http://codemirror.net/LICENSE
+
+(function (mod) {
+  "use strict";
```

Comment:
You want to put the `"use strict"` in the main body — here it'll only affect the module header.